<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Variablemonitor</class>
 <widget class="QDialog" name="Variablemonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Variable Monitor</string>
  </property>
  <widget class="QTableWidget" name="List_scan">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>261</width>
     <height>541</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>40</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Prior</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Start</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>End</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Step</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="List_calculate">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>271</width>
     <height>541</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>134</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Formula</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="List_calibrate">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>40</y>
     <width>261</width>
     <height>541</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>80</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>File</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Fit</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="Vari_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Vari_add">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add New Variable</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Vari_type">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Fixed</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Scan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Calculated</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Data File</string>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="List_fix">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>131</width>
     <height>541</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="Import">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Import Variables</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Search">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
